src/components/User.vue

```diff
@@ -1,38 +1,122 @@
-<script setup lang="ts">
-  import { createApp } from 'vue';
+<script lang="ts">
+import { defineComponent, ref, watchEffect } from 'vue';
+import Search from './Search.vue';
+import InfoBlock from './InfoBlock.vue';
+import Avatar from './Avatar.vue';
+import ProfileInfo from './ProfileInfo.vue';
+import Bio from './Bio.vue';
+import Stats from './Stats.vue';
+import StatItem from './StatItem.vue';
 
+interface UserInfo {
+  avatar_url: string;
+  name: string | null;
+  login: string;
+  created_at: string;
+  bio: string | null;
+  public_repos: number;
+  followers: number;
+  following: number;
+  location: string | null;
+  blog: string | null;
+  twitter_username: string | null;
+  company: string | null;
```

Comment:
With optional values, it should be for example: `company?: string`
https://www.typescriptlang.org/docs/handbook/interfaces.html#optional-properties